worlds/stardew_valley/__init__.py

```diff
@@ -267,7 +267,12 @@ def generate_basic(self):
         pass
 
     def get_filler_item_name(self) -> str:
-        return "Joja Cola"
+        include_traps = self.options[options.TrapItems] != options.TrapItems.option_no_traps
```

Comment:
This is an incredibly costly operation for how much this function can potentially get called. Should definitely cache the results. You inspired me to finally get around to doing this so example here: https://github.com/ArchipelagoMW/Archipelago/pull/2070